tests/integration/cattletest/core/test_svc_discovery.py

```diff
@@ -789,6 +789,82 @@ def test_remove_environment_w_active_svcs(super_client,
     assert instance_service_map[0].state == 'removed'
 
 
+def _validate_compose_instance_start(service, super_client, number):
+    instances = super_client. \
+        list_container(name="compose_" + service.name
+                            + "_" + number, state="running")
+    assert len(instances) == 1
+    return instances[0]
+
+
+def _validate_instance_removed(service, super_client, number):
+    instances = super_client. \
+        list_container(name="compose_" + service.name + "_" + number)
+    assert len(instances) == 1
+    instance = instances[0]
+    wait_for_condition(
+        super_client, instance, _resource_is_removed,
+        lambda x: 'State is: ' + x.state)
+
+
+def _resource_is_removed(resource):
+    return resource.state == 'removed'
+
+
+def test_valiate_service_scaleup_scaledown(super_client,
+                                           admin_client, sim_context, nsp):
+    env = admin_client.create_environment(name="compose")
+    env = admin_client.wait_success(env)
+    assert env.state == "active"
+
+    image_uuid = sim_context['imageUuid']
+    launch_config = {"imageUuid": image_uuid}
+
+    service = super_client.create_service(name=random_str(),
```

Comment:
@cjellick Because there are certain parameters that I don't expose to an end user, and that are needed in order to run against simulator. Like networkId, agent URI predicate, etc